src/ansys/simai/core/utils/configuration.py

```diff
@@ -103,3 +131,22 @@ def _auth_hash(self) -> str:
         if config_file_profile:
             hasher.update(config_file_profile.encode())
         return hasher.hexdigest()
+
+    @field_validator("organization", mode="before")
+    @classmethod
+    def check_organization_exists(cls, val, info):
```

Comment:
Would be nice to type `info` where it's used (the type is `pydantic.ValidationInfo`)

Reply:
do you mean to declare the type of `info` in the argument annotation?

Reply:
Yes, but using `from pydantic import ValidationInfo`, `info: ValidationInfo` (ofc)

Reply:
ofc